src/views/v2/commerce/gateway/stripe/card_element.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Buttons for Stripe
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/gateway/stripe/buttons.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ * @var bool $must_login [Global] Whether login is required to buy tickets or not.
+ */
+
+if ( $must_login ) {
```

Comment:
```suggestion
if ( ! empty( $must_login ) ) {
```
Let's have it this way so it's in line with what we have on the other templates and it's safer.